extensions/smallrye-reactive-messaging/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/deployment/SmallRyeReactiveMessagingProcessor.java

```diff
@@ -91,8 +118,8 @@ void validateBeanDeployment(
 
         for (InjectionPointInfo injectionPoint : validationPhase.getContext()
                 .get(BuildExtension.Key.INJECTION_POINTS)) {
-            if (injectionPoint.getRequiredType().name().equals(NAME_EMITTER)) {
-                AnnotationInstance stream = injectionPoint.getRequiredQualifier(NAME_STREAM);
+            if (injectionPoint.getRequiredType().name().equals(EMITTER)) {
+                AnnotationInstance stream = injectionPoint.getRequiredQualifier(STREAM);
```

Comment:
1.0.4 replaces @Stream with @Channel (@Stream is deprecated but still supported).

Reply:
Thanks for pointing that out!